tests/rptest/tests/flink_basic_test.py

```diff
@@ -40,10 +42,6 @@ def __init__(self, test_context, *args, **kwargs):
 
         return
 
-    def tearDown(self):
```

Comment:
Why is this `tearDown` no longer needed?

Reply:
Because we do not need to delete topics in between tests anymore. They just deleted along with RP by service cleaning. We do needed that in scale tests which was scrapped by core team as "too complicated" in November, 2023 :)